### PR TITLE
Last fixes for next v4 update

### PR DESCRIPTION
**Pull request by Amerlander**
- change settings for Calliope mini 2 template to work with radio extension
- updates to make sure the state of bluetooth is reflected in the user settings when the radio extension gets loaded

### CODE REVIEW

**Comment by Amerlander** [comment]
@abchatra Can you please review and merge this pull request? Once merged, please bump a new version and release it as the stable version.

I'm currently working on porting to PXT v9.0.11, so this might be the last update for the v4 stream.

Thank you!

**Review by abchatra — APPROVED** [review]

**Comment by abchatra** [comment]
Done

**Pull request merged**
